config/manifests/bases/awx-resource-operator.clusterserviceversion.yaml

```
@@ -448,6 +448,20 @@ spec:
         path: instance_groups
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:text
+      - displayName: Inventory State
+        path: state
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:select:present
+        - urn:alm:descriptor:com.tectonic.ui:select:absent
+        - urn:alm:descriptor:com.tectonic.ui:select:exists
+      - displayName: Inventory Variables
+        path: variables
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - displayName: Authentication Secret
+        path: connection_secret
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
```

Could you add this so that in the UI the secrets will populate in a drop-down menu?

```suggestion
        - urn:alm:descriptor:com.tectonic.ui:text
        - urn:alm:descriptor:io.kubernetes:Secret
```

---

We should really have this on all connection_secret parameters in the ClusterServiceVersion file. That shouldn't block this PR merging.